src/service/xhr-impl.js

```diff
@@ -195,9 +176,11 @@ export class Xhr {
    *
    * @param {string} input
    * @param {?FetchInitDef=} opt_init
+   * @param {boolean=} opt_skipProcessing Disables any processing of the
+   *     response, including asserting the response is 2XX.
    * @return {!Promise<!JSONType>}
    */
-  fetchJson(input, opt_init) {
+  fetchJson(input, opt_init, opt_skipProcessing) {
```

Comment:
Ok. Let's then go ahead and switch API to always return the `Response`. I'm happy with this - it gets us very close to the actual `fetch`.